packages/lesswrong/components/messaging/ConversationContents.tsx

```diff
@@ -24,7 +24,7 @@ const styles = (theme: ThemeType): JssStyles => ({
       // on mobile. This fixes that.
       display: "flex",
     },
-    ...(isEAForum && {
+    ...(isFriendlyUI && {
```

Comment:
Plausibly this should just be default btw - that separation between text field and messages is helpful/nice to have